src/App.tsx

```diff
@@ -140,25 +140,7 @@ export default function App(): JSX.Element {
                     <HaukiNavigation />
                     <Main id="main">
                       <CreateNewOpeningPeriodPage
-                        resourceId={match.params.id}
```

Comment:
Looks like I removed this as well in this PR. We do not need the exception page anymore.